packages/docusaurus-plugin-content-docs/src/options.ts

```diff
@@ -168,11 +167,5 @@ export function validateOptions({
 
   const normalizedOptions = validate(OptionsSchema, options) as PluginOptions;
 
-  if (normalizedOptions.admonitions) {
```

Comment:
this is removed, but admonitions remains in schema + default value

We want to still allow custom admonitions? Are custom admonitions configured globally or per plugin?

Do we want to do a breaking change and add a custom error message, or try to keep retro-compatibility?

Reply:
I think we should keep this here for the time being, but when we eventually have a centralized `markdown.remarkPlugins` config (#5999) I'd like to move it there. Note that I also want to have `admonitions` as part of the default MDX loader preset, so it can be configured as part of the `docusaurus-remark-preset`.

Reply:
@Josh-Cena I agree we should keep the options there, at least for now

The problem is that the provided options do not make sense anymore

https://github.com/lorikrell/gamerchic/blob/ec6e1595a51b7e65538262dd9308fb7517e42f9f/docusaurus.config.js#L359

```js
{
        admonitions: {
          customTypes: {
            homebrew: {
              keyword: `homebrew`,
              infima: true,
              svg: '<svg /></svg>'
            },
            gamerchic: {
              keyword: `gamerchic`,
              infima: true,
              svg: '<svg /></svg>'
            },
          },
        },
}  
```

Now the options should look like:

```js
{
        admonitions: {
          customKeywords: ["homebrew","gamerchic"]
        },
}  
```

And custom styling/svg should be applied by swizzling the admonition component.


@lex111 it's worth supporting that to ensure that the above site config have a way to upgrade Docusaurus without losing its custom admonitions